src/firefly/js/visualize/ChartsTableViewPanel.jsx

```diff
@@ -52,11 +56,11 @@ var ChartsPanel = React.createClass({
 
     getInitialState() {
         return {
-            chartType : SCATTER,
-            optionsShown : true,
+            chartType: SCATTER,
+            optionsShown: true,
             widthPx: 700,
             heightPx: 300,
-            debouncedResize: debounce(this.onResize, 500, {'leading':false, 'trailing':true})};
+            debouncedResize: debounce(this.onResize, 200, {'leading':false, 'trailing':true, 'maxWait':10000})};
```

Comment:
Also- after thinking about it I don't think you need a max wait.  The use case would be someone holding the mouse a resizing the component for 10 seconds with out letting up.  It that is what they want to do then just let me.